lib/exd/escript/main.ex

```diff
@@ -115,8 +121,15 @@ available apis:
 
   defp payload(payload, "native") do
     splited = Enum.map(payload, &String.split(&1, ":"))
```

Comment:
Use simple: String.split(payload, ":", parts: 2)

Than you doesn't need to rewrite the last part of a function.